study.md

```diff
@@ -34,7 +34,8 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+<!-- The two I noticed and picked out were map and drop. I assume that the only real difference between drop on the number or array would be the values that it is working with. Map is similar except it returns an enumerable.
```

Comment:
This does not answer the question.

`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.